syntax = "proto3";

option go_package = '../proto';

import "google/protobuf/empty.proto";

service ResumeService{
  // 获取简历列表
  rpc GetResumeList(GetResumeListRequest) returns (GetResumeListResponse);

  // 创建简历
  rpc CreateResume(CreateResumeRequest) returns ();

  // 更新简历
  rpc UpdateResume(UpdateResumeRequest) returns (google.protobuf.Empty);

  // 删除简历
  rpc DeleteResume(DeleteResumeRequest) returns(google.protobuf.Empty);
}


message GetResumeListRequest{

}

message GetResumeListResponse{

}

message CreateResumeRequest{

}

message UpdateResumeRequest{

}

message DeleteResumeRequest{

}
